code/modules/cargo/packs.dm

```diff
@@ -2439,6 +2439,31 @@
 					/obj/item/clothing/head/helmet/bluetaghelm)
 	crate_name = "laser tag crate"
 
+/datum/supply_pack/costumes_toys/knucklebone
+	name = "Knucklebone Game Crate"
```

Comment:
Knucklebones with an S at the end

Reply:
The name of the game is Knucklebone dou.